lab/name.js

```diff
@@ -0,0 +1,6 @@
+const getName = () => {
```

Comment:
With this function we are wanting to destructor name off of the obj. Right now if you feed it a obj with the name of anything other then pink if would fail we want it to pass no matter what name is in the obj. ex.                    `const getProp = ({property}) => {
return property};` like so if you need help let us know in the ta-station

Reply:
Thank you! I will get it done!

Reply:
anytime